mir-ci/mir_ci/robot_libraries/WaylandHid.py

```diff
@@ -21,6 +21,7 @@ class WaylandHid(VirtualPointer):
 
     def __init__(self) -> None:
         self.ROBOT_LIBRARY_LISTENER = self
+        self.pointer_position = {"x": 0.0, "y": 0.0}
```

Comment:
Why do we deal with floats for x/y, Wayland [only does](https://wayland.app/protocols/wlr-virtual-pointer-unstable-v1#zwlr_virtual_pointer_v1:request:motion_absolute) int?

Also, since this is internal, let's prefix with `_`? On one hand a tuple could do, but then we're returning it as a dict often enough that it's probably appropriate to be a dict.